### PR TITLE
fix #279671 - MusicXML export loses tablature info, produces unrecogn…

### PR DESCRIPTION
**Pull request by lvinken**
…izable notation

### CODE REVIEW

File: mscore/exportxml.cpp [hunk 1]
```diff
@@ -4729,6 +4729,58 @@ void ExportMusicXml::print(Measure* m, int idx, int staffCount, int staves)
 
       }
 
+//---------------------------------------------------------
+//  findAndExportTbdClef
```
Comment (anatoly-os) [inline mscore/exportxml.cpp]: @lvinken could you please fix the comment? Should be `exportDefaultClef`.

Comment (lvinken) [reply mscore/exportxml.cpp]: Done

Comment (anatoly-os) [reply mscore/exportxml.cpp]: and squash the commits like `git rebase -i HEAD~2` and `git push -f origin 279671-default-clef-musicxml-export`?

Comment (lvinken) [reply mscore/exportxml.cpp]: Also done

**Pull request merged**